projectgenerator/libili2pg/iliimporter.py

```diff
@@ -121,7 +121,9 @@ def run(self):
             # By default try JAVA_HOME and PATH
             java_paths = []
             if 'JAVA_HOME' in os.environ:
-                java_paths += [os.path.join(os.environ['JAVA_HOME'], 'java')]
+                paths = os.environ['JAVA_HOME'].split(";")
```

Comment:
Should that be [`os.pathsep`](https://docs.python.org/3/library/os.html#os.pathsep)?